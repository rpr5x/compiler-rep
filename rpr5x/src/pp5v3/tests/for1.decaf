void main() {
  int i;
   for (i = 10; i > 0; i = i - 2)
     Print(i);
  Print("after ", i);
  i = 0;
  for (; true; ) {
	i = i + 1;
      if (i == 5) break;
  }
  Print("done", i);
}