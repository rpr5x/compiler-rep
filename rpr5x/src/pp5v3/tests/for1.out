Loaded: /if14/ww6r/WorkingDisk/TA/compiler/exceptions/exceptions.s
108642after 0done5